backend/tests/apps/core/api/status_test.py

```
@@ -0,0 +1,43 @@
+import json
+from unittest.mock import Mock
+
+import pytest
+import requests
+from django.conf import settings
+
+from apps.core.api.status import status_view
```

Pylint (3.3.7)

[error] No name 'status_view' in module 'apps.core.api.status'

(E0611)